libraries/AP_HAL_SITL/sitl_rangefinder.cpp

```diff
@@ -32,7 +32,7 @@ float SITL_State::_sonar_pin_voltage() const
     }
 
     const float altitude = sitl_model->rangefinder_range();
-    if (altitude == INFINITY) {
+    if (is_equal(altitude, INFINITY)) {
```

Comment:
```suggestion
    if (!isfinite(altitude) {
```

Reply:
LOL two suggestions at the same time go with isinf()

Reply:
ha! But who's *more* right? :)

Reply:
Which ever he chooses?? Since either is good enough :)

Reply:
`INFINITY` is a specific bit pattern.  You're not kinda-sorta infininite - so `is_equal`'s epsilon check isn't the right thing to do.